galleon-feature-pack/src/main/resources/modules/system/layers/base/org/jboss/resteasy/resteasy-spring/main/module.xml

```diff
@@ -28,5 +28,12 @@
     </resources>
 
     <dependencies>
+        <module name="javax.ws.rs.api"/>
```

Comment:
@jamezp this GAV is different from the one in `pom.xml`:

```xml
 <dependency>
            <groupId>org.jboss.spec.javax.ws.rs</groupId>
            <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
        </dependency>
```

Will it cause any problem?

Reply:
No, it shouldn't matter. I personally prefer using the Jakarta EE version as we only fork because we need runtime changes for things like running with a security manager.